editor/js/Config.js

```diff
@@ -19,7 +19,12 @@ var Config = function ( name ) {
 
 		'project/vr': false,
 
-		'settings/history': false
+		'settings/history': false,
+
+		'controls/translate': 'w',
```

Comment:
Do you mind renaming these to `settings/controls/translate`...?

Reply:
Done!